org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/server/RemoteSonarProject.java

```diff
@@ -67,26 +57,6 @@ public void setName(String name) {
     this.name = name;
   }
 
-  public String asString() {
```

Comment:
Is this a behavior change? Or is this dead code?

Reply:
Was dead code, I did some cleanup because some Javadoc errors on this class were preventing the build. I could have extracted that in a separate PR... but let say this is Fall cleanup ;)